app/react/portainer/environments/wizard/EnvironmentsCreationView/WizardDocker/APITab/index.ts

```diff
@@ -1 +1,3 @@
 export { APITab } from './APITab';
+export { TLSFieldset, tlsConfigValidation } from './TLSFieldset';
+export * from './types';
```

Comment:
please try not to export * or import *, but instead exporting or importing exactly what you need

Reply:
fixed